web/client/components/maps/MapCard.jsx

```diff
@@ -87,7 +87,16 @@ class MapCard extends React.Component {
     getCardStyle = () => {
         if (this.props.map.thumbnail) {
             return assign({}, this.props.style, {
-                backgroundImage: 'linear-gradient(rgba(0, 0, 0, ' + this.props.backgroundOpacityStart + '), rgba(0, 0, 0, ' + this.props.backgroundOpacityEnd + ') ), url(' + (this.props.map.thumbnail === null || this.props.map.thumbnail === "NODATA" ? thumbUrl : decodeURIComponent(this.props.map.thumbnail)) + ')'
+                backgroundImage: 'linear-gradient(rgba(0, 0, 0, '
```

Comment:
Are these only reformats? I can see similar ones in following files, so I need to know if I can just ignore them.

Reply:
Yes, only to add the comment to the various decodeURIComponent